### PR TITLE
:basecamp: Section 2: Fundamentals

### PR DESCRIPTION
**Pull request by Palwisha-18**

### CODE REVIEW

File: 02_Fundamentals/1_JavaScript_Syntax/learning_outcome.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- Statements
+    * are used to declare a variable or to instruct the JavaScript engine to do a task.
+    * are terminated by a semicolon (`;`).
+
+- Blocks
+    * curly braces (`{}`) are used to form a block.
+    * it contains zero or more statements.
+
+- Identifers
```
Comment (ahmadjanan) [inline 02_Fundamentals/1_JavaScript_Syntax/learning_outcome.md]: Identifers -> Identifiers

File: 02_Fundamentals/2_Variables/learning_outcome.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h3>JavaScript Variables:</h3>
+
+- variables are used to store values and are case-sensitive. Rules of identifers apply to variables.
```
Comment (ahmadjanan) [inline 02_Fundamentals/2_Variables/learning_outcome.md]: identifers -> identifiers

File: 02_Fundamentals/2_Variables/learning_outcome.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- `var` keyword is used to declare/initalize the variable.
```
Comment (ahmadjanan) [inline 02_Fundamentals/2_Variables/learning_outcome.md]: initalize -> initialize

File: 02_Fundamentals/3_Data_Types/learning_outcome.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<h3>JavaScript Data Types:</h3>
+
+JavaScript has two types of data types: Primitive and Complex.
```
Comment (ahmadjanan) [inline 02_Fundamentals/3_Data_Types/learning_outcome.md]: two types of data types -> two data types

File: 02_Fundamentals/3_Data_Types/learning_outcome.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    * JavaScript defines that `null` is equal to `undefined`.
+- number
+    * used to represent both integer and floating-point numbers.
+- NaN
+    * stands for Not a Number.
+    * any operation with NaN returns NaN.
+    * the NaN does not equal any value, including itself.
+- string
+    * a sequence of zero or more characters. It begins and ends with either a single quote(`'`) or a double quote (`"`).
+    * they are immutable in JavaScript. e.g, characters of an existing string cannot be chamged
```
Comment (ahmadjanan) [inline 02_Fundamentals/3_Data_Types/learning_outcome.md]: chamged -> changed

**Review by ahmadjanan — CHANGES_REQUESTED** [review]

**Comment by ahmadjanan** [comment]
**Suggestion**: Let's rename all the `learning_outcome.md` -> `readme.md`, so they get displayed in their respective git directories and can be read without having to be opened.

File: 02_Fundamentals/1_JavaScript_Syntax/learning_outcome.md (hunk 1 above)
Comment (Palwisha-18) [reply 02_Fundamentals/1_JavaScript_Syntax/learning_outcome.md]: fixed.

File: 02_Fundamentals/2_Variables/learning_outcome.md (hunk 2 above)
Comment (Palwisha-18) [reply 02_Fundamentals/2_Variables/learning_outcome.md]: fixed.

File: 02_Fundamentals/2_Variables/learning_outcome.md (hunk 3 above)
Comment (Palwisha-18) [reply 02_Fundamentals/2_Variables/learning_outcome.md]: fixed.

File: 02_Fundamentals/3_Data_Types/learning_outcome.md (hunk 4 above)
Comment (Palwisha-18) [reply 02_Fundamentals/3_Data_Types/learning_outcome.md]: fixed.

File: 02_Fundamentals/3_Data_Types/learning_outcome.md (hunk 5 above)
Comment (Palwisha-18) [reply 02_Fundamentals/3_Data_Types/learning_outcome.md]: fixed.

**Comment by Palwisha-18** [comment]
> **Suggestion**: Let's rename all the `learning_outcome.md` -> `readme.md`, so they get displayed in their respective git directories and can be read without having to be opened.

Changed. I had the same thing in mind. Good suggestion.

**Review by ahmadjanan — APPROVED** [review]
LGTM 👍

**Pull request merged**